CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+👋 Hello!
+
+Thanks for helping out in the [source map effort](https://ecma-international.org/task-groups/tc39-tg4/). The following guide is meant to help you get to know our workstreams, meetings, and plans.
+
+### Joining Us!
```

Comment:
nit: markdown headings are not normalized.  there should be a `#` at the start of the document and subsequent headings should follow hierarchical structure (`##`, `###`, ...)

Reply:
Thanks! WIll update!